jgiven-core/src/test/java/com/tngtech/jgiven/report/asciidoc/AsciiDocReportConfigTest.java

```diff
@@ -0,0 +1,37 @@
+package com.tngtech.jgiven.report.asciidoc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class AsciiDocReportConfigTest {
```

Comment:
How about another test with an input "--sourceDir=". We currently fail gracefully and print the help message, which I think is good behavior.

Reply:
The `ConfigOptionParser` calls `System.exit()` in this case and currently I'm not sure how to test for that case without introducing a lot of additional complexity.